### ISSUE TITLE
total activity time is difficult to read in statistics

### ISSUE THREAD

**Issue by ja7833** [ramack/ActivityDiary#264]
I discovered this App via F-Droid. I like the interface which is easy to understand. However, in statistics I wish the total activity time was displayed above or below the doughnut chart in large text instead of showing the totals on the graph which are too small to be legible.

![2019-04-26](https://user-images.githubusercontent.com/50032102/56836971-d01be800-6846-11e9-821a-57c242e005ca.png)

**Comment by ramack** [comment]
You're totally right, currently it is close to unreadable. But I fear, that the available space is not sufficient to display all activities in a list beside the the pie chart.

What do you think about providing the list in another view, maybe reachable via swipe left/right or in some form of tabbed view as an alternative to the pie chart?

**Comment by ja7833** [comment]
I was thinking to only show the highlighted element of the doughnut chart.

A straight list (a bar chart might make the text large enough) with an ascending and descending sort as an option would also be welcome.
